Functions.Templates/Templates/BlobTrigger-PowerShell/function.json

```diff
@@ -0,0 +1,12 @@
+{
+    "disabled": false,
+    "bindings": [
+        {
+            "name": "myBlob",

```

Comment:
Why not just `Blob`?  The name ends up being a PowerShell parameter which is Pascal cased by convention

Reply:
if there is an blob input binding, function.json would have
```
"type" : "blob"
```
It would be confusing to use Blob for name. InputBlob?

Reply:
For a input binding, the type is `"blobTrigger"`, so it's OK to use `Blob` as the name.
But it would be confusing in a blob output binding.